{"name": "测试数据：【 1532341，111111a，{except_result}，10000，token】", "status": "broken", "statusDetails": {"message": "TypeError: post() got multiple values for keyword argument 'verify'", "trace": "self = <testcases.api_test.test_03_login.TestUserLogin object at 0x000001AE94370390>\nusername = '1532341', password = '111111a', login_type = 1, except_code = 10000\nexcept_msg = 'token'\n\n    @allure.story(\"用例--登录用户\")\n    @allure.description(\"该用例是针对获取用户登录接口的测试\")\n    @allure.issue(\"https://www.cnblogs.com/wintest\", name=\"点击，跳转到对应BUG的链接地址\")\n    @allure.testcase(\"https://www.cnblogs.com/wintest\", name=\"点击，跳转到对应用例的链接地址\")\n    @allure.title(\"测试数据：【 {username}，{password}，{except_result}，{except_code}，{except_msg}】\")\n    @pytest.mark.single\n    @pytest.mark.parametrize(\"username, password, login_type, except_code, except_msg\",\n                             api_data[\"test_login_user\"])\n    def test_login_user(self, username, password, login_type, except_code, except_msg):\n        logger.info(\"*************** 开始执行用例 ***************\")\n>       result = login_user(username, password)\n\ntest_03_login.py:28: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n..\\..\\operation\\user.py:90: in login_user\n    res = user.login(data=payload, headers=header)\n..\\..\\api\\user.py:26: in login\n    return self.post(\"/idserverhq/public/auth/login\", **kwargs)\n..\\..\\core\\rest_client.py:17: in post\n    return self.request(url, \"POST\", data, json, **kwargs,verify=False)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <api.user.User object at 0x000001AE93AB95C0>\nurl = 'https://dev2-hq-kong.tiens.com/idserverhq/public/auth/login/idserverhq/public/auth/login'\nmethod = 'POST', data = {'password': '111111a', 'username': '1532341'}\njson = None\nkwargs = {'headers': {'Content-Type': 'application/x-www-form-urlencoded'}, 'verify': False}\nheaders = {'Content-Type': 'application/x-www-form-urlencoded'}, params = None\nfiles = None, cookies = None\n\n    def request(self, url, method, data=None, json=None, **kwargs):\n        # url = self.api_root_url + url\n        headers = dict(**kwargs).get(\"headers\")\n        params = dict(**kwargs).get(\"params\")\n        files = dict(**kwargs).get(\"params\")\n        cookies = dict(**kwargs).get(\"params\")\n        if url == \"/idserverhq/public/auth/login\":\n            url = self.api_root_login_url + url\n        else:\n            url = self.api_root_url + url\n        self.request_log(url, method, data, json, params, headers, files, cookies)\n        if method == \"GET\":\n            return self.session.get(url, **kwargs,verify=False)\n        if method == \"POST\":\n>           return requests.post(url, data, json, **kwargs,verify=False)\nE           TypeError: post() got multiple values for keyword argument 'verify'\n\n..\\..\\core\\rest_client.py:42: TypeError"}, "description": "该用例是针对获取用户登录接口的测试", "attachments": [{"name": "log", "source": "1dfee1b5-9751-4ec5-a158-8a78b5b1e593-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "username", "value": "'1532341'"}, {"name": "password", "value": "'111111a'"}, {"name": "login_type", "value": "1"}, {"name": "except_code", "value": "10000"}, {"name": "except_msg", "value": "'token'"}], "start": 1669177922899, "stop": 1669177922901, "uuid": "d31b1cc0-29d2-4629-b739-4418a93672ce", "historyId": "fc744133cd1f19c331d93737caf7e646", "testCaseId": "7272db244dabef390096479ae3196642", "fullName": "testcases.api_test.test_03_login.TestUserLogin#test_login_user", "labels": [{"name": "story", "value": "用例--登录用户"}, {"name": "severity", "value": "normal"}, {"name": "feature", "value": "用户登录模块"}, {"name": "epic", "value": "针对单个接口的测试"}, {"name": "tag", "value": "single"}, {"name": "parentSuite", "value": "testcases.api_test"}, {"name": "suite", "value": "test_03_login"}, {"name": "subSuite", "value": "TestUserLogin"}, {"name": "host", "value": "lijuan"}, {"name": "thread", "value": "19932-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.api_test.test_03_login"}], "links": [{"type": "test_case", "url": "https://www.cnblogs.com/wintest", "name": "点击，跳转到对应用例的链接地址"}, {"type": "issue", "url": "https://www.cnblogs.com/wintest", "name": "点击，跳转到对应BUG的链接地址"}]}